docs/source/user/versions.rst

```diff
@@ -0,0 +1,29 @@
+
+Versions Supported in Bioconda
+==============================
+Bioconda only supports some versions of conda packages.
+Here is an incomplete list of the dependencies that might be relevant to your
+conda environment.
+
+Operating Systems
+-----------------
+Bioconda only supports 64-bit Linux and Mac OS
+
+Python
+------
+Bioconda only supports python 3.6 and 3.7. The community is currently working on adding support for
```

Comment:
And 2.7. Can you mention that this is only relevant for architecture-specific builds? `noarch: python` packages are unaffected by these restrictions.

Reply:
Good catch r.e. 2.7. But [my recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/trtools/meta.yaml) is `noarch: python` but I'm still having these issues. Any idea why? Is it because of some of my dependencies?

Reply:
That must be due to some dependencies. For `noarch` packages the pinnings (what underlies the python versions we build for) don't play a role.

Reply:
I added a note with those details. Is it accurate and reasonable comprehensible?

Reply:
Yeah, that's a fine note, thanks!